package.json

```diff
@@ -5,13 +5,15 @@
   "author": "central419",
   "type": "module",
   "scripts": {
-    "action": "deno run -A ./run/action.ts",
-    "check": "deno run -A ./run/check.ts",
-    "verify": "deno check **/*.ts",
+    "action": "bun run run-deno ./run/action.ts",
+    "check": "bun run run-deno ./run/check.ts",
     "coverage": "deno coverage coverage/profile --ignore=node_modules",
     "knip": "knip",
-    "progress": "deno run -A ./run/progress.ts",
-    "test": "deno test -A"
+    "lint": "deno lint",
+    "progress": "bun run run-deno ./run/progress.ts",
+    "run-deno": "deno run --allow-read --allow-env --allow-net=$(paste -d, -s ./allowed_domains)  --allow-write=$(paste -d, -s ./allowed_paths)",
```

Comment:
I didn't want to have a hardcoded list of paths & nets to allow in each script.
Having a separate file for allowed domains and paths allowed to write to seems extremely handy for review later, as this list may grow.

I do wonder if the `paste` library is available on all platforms.